.github/workflows/bump-backend-version.yml

```diff
@@ -0,0 +1,36 @@
+name: Bump backend version on merge
+on:
+    push:
+        branches:
+            - development
+        paths:
+            - 'backend/docs/**'
+            - 'backend/reports/**'
+            - 'backend/src/**'
+            - 'backend/templates/**'
+            - 'backend/test/**'
+
+jobs:
+  bump-version:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 18.14
```

Comment:
@ASpiteri-BCGov we have to figure out how not to hard code this version every time, @kshapka-bcgov maybe variables/secret within github... this might be future ticket considerations.  Although it would be nice if this version came from the package.json some how...

Reply:
Although I was going to include this, as we discussed in the meeting, for MVP I removed all considerations of examining the existing package.json file.